Mathlib/Tactic/Replace.lean

```diff
@@ -11,7 +11,6 @@ namespace Mathlib.Tactic
 open Lean Elab.Tactic
 
 syntax "replace " haveDecl : tactic
```

Comment:
I realize it's not actually added in this PR, but could you also add a docstring or copy paste the one from `replace'` here?

Reply:
Sure -- done!